docs/_deployments/ja_aws-lambda.md

```diff
@@ -0,0 +1,342 @@
+  signingSecret: process.env.SLACK_SIGNING_SECRET,
+  // `processBeforeResponse` オプションは、あらゆる FaaS 環境で必須です。
+  // このオプションにより、Bolt フレームワークが `ack()` などでリクエストへの応答を返す前に
+  // `app.message` などのメソッドが Slack からのリクエストを処理できるようになります。FaaS では
+  // 応答を返した後にハンドラーがただちに終了してしまうため、このオプションの指定が重要になります。
+  processBeforeResponse: true
+});
+
+// ボットトークンと、AWS Lambda に対応させたレシーバーを使ってアプリを初期化します。
+const app = new App({
+  token: process.env.SLACK_BOT_TOKEN,
+  receiver: expressReceiver
+});
+
+// Bolt の ExpressReceiver を使って AWSServerlessExpress のサーバーを初期化します。
+const server = awsServerlessExpress.createServer(expressReceiver.app);
+```
+
+最後に、アプリのソースコードの末尾にある [HTTP サーバーを起動する部分](https://github.com/slackapi/bolt-js-getting-started-app/blob/main/app.js#L40-L45)を編集して、AWS Lambda 関数のイベントに応答するようにします。
+
+```javascript
+// Lambda 関数のイベントを処理します
+module.exports.handler = (event, context) => {
+  console.log('⚡️ Bolt app is running!');
+  awsServerlessExpress.proxy(server, event, context);
+};
+```
+
+完成したアプリのソースコードは、⚡️[deploy-aws-lambda][deploy-aws-lambda-app/app.js] のサンプルのようになります。
+
+**2. serverless.yml を追加する**
+
+Serverless Framework のプロジェクトでは、アプリの設定とデプロイに `serverless.yml` ファイルを使用します。
+
+アプリのルートディレクトリに `serverless.yml` という名前のファイルを新規作成し、次の内容を貼りつけます。
+
+```yaml
+service: serverless-bolt-js
+frameworkVersion:'2'
+provider:
+  name: aws
+  runtime: nodejs12.x
+  environment:
+    SLACK_SIGNING_SECRET: ${env:SLACK_SIGNING_SECRET}
+    SLACK_BOT_TOKEN: ${env:SLACK_BOT_TOKEN}
+functions:
+  slack:
+    handler: app.handler
+    events:
+      - http:
+          path: slack/events
+          method: post
+plugins:
+  - serverless-offline
+```
+
+> 💡 `SLACK_SIGNING_SECRET` と `SLACK_BOT_TOKEN` の環境変数は、ローカルマシンで設定しておく必要があります。[Slack の環境変数をエクスポートする方法](/bolt-js/tutorial/getting-started#setting-up-your-local-project)を入門ガイドで参照してください。
+
+**3. serverless-offline モジュールをインストールする**
+
+ローカルでの開発を容易にするため、`serverless-offline` モジュールを使ってデプロイ対象の関数をエミュレートできるようにしましょう。
+
+次のコマンドを実行して、開発用の devDependencies としてインストールします。
+
+```bash
+npm install --save-dev serverless-offline
+```
+
+これで Bolt アプリを AWS Lambda と Serverless に対応させることができました。次はアプリの実行とデプロイに進みます。
+
+---
+
+### アプリをローカルで実行する
+
+アプリを AWS Lambda 関数に応答させるための準備が完了したので、次にローカルでアプリを実行できるように環境を設定します。
+
+**1. ローカルのサーバーを起動する**
+
+まず、AWS Lambda 関数のイベントをリッスンするため、`serverless offline` コマンドを実行します。
+
+```zsh
+npx serverless offline --noPrependStageInUrl
+```
+
+次に、ngrok を使って Slack のイベントをローカルマシンに転送します。
+
+```zsh
+ngrok http 3000
+```
+
+> 💡 パブリック URL の作成方法と、ローカルマシンへのリクエストの転送方法については、[ngrok の使い方][getting-started-guide-setting-up-events]を参照してください。
+
+**2. リクエスト URL を変更する**
+
+次に、[Slack アプリの設定](https://api.slack.com/apps)を開き、**リクエスト URL** を ngrok のウェブアドレスに変更します。
+
+> 💡 **リクエスト URL** は `/slack/events` で終わる文字列で、例えば `https://abc123.ngrok.io/slack/events` のようになります。
+
+まず、サイドバーの「**Interactivity & Shortcuts**」を選択し、**リクエスト URL** を更新します。
+
+![「Interactivity & Shortcuts」ページ](../assets/interactivity-and-shortcuts-page.png "「Interactivity & Shortcuts」ページ")
```

Comment:
@seratch do we want to include translated images as well? Or, is Slack's App Settings page only available in english?

Reply:
The admin pages are only in English. Therefore, we can reuse the screenshots.

Reply:
Ah cool, good to know!